### DISCUSSION TITLE
Failed to install and or run

### DISCUSSION THREAD

**Discussion by getlocaladmin** [MustardChef/WSABuilds#202] [Q&A]
Any idea why this is happening? I am able to install WSA on all of my computers but on this surface laptop studio 2 It keeps on giving this error.

<img width="837" alt="image" src="https://github.com/MustardChef/WSABuilds/assets/113044451/cdc89bda-5532-4ee9-88ea-372df58c91c1">

**Comment by MustardChef** [comment]
@getlocaladmin 
Remove the previously installed version by running the command below on Terminal/Powershell (Admin):
```powershell
Get-AppxPackage -Name "MicrosoftCorporationII.WindowsSubsystemForAndroid" -AllUsers | Remove-AppxPackage -AllUsers
```

**Reply by Marlock1234** [reply]
Hi, I have tried this but it does not work. I am still not able to install the app. Any other ideas?

**Reply by MustardChef** [reply]
Same error as above?

**Comment by Marlock1234** [comment]
No. I am getting this: [](url)
![1](https://github.com/MustardChef/WSABuilds/assets/100565941/7917d02a-3085-4de5-be72-01a90f176ee8)

**Comment by DJayparmar** [comment]
![image](https://github.com/MustardChef/WSABuilds/assets/24941442/8297a6c6-b90b-4630-bc66-e70c2095668e)

**Comment by MustardChef** [comment]
If anyone has this same issue, please open an issue and provide full system information.
